### PR TITLE
Allow multiple UPSTREAM_WORKSPACE http* entries.

### PR DESCRIPTION
**Pull request by 130s**

### CODE REVIEW

File: travis.sh [hunk 1]
```diff
@@ -93,7 +93,11 @@ case "$UPSTREAM_WORKSPACE" in
         ;;
     http://* | https://*) # When UPSTREAM_WORKSPACE is an http url, use it directly
         travis_run wstool init .
-        travis_run wstool merge $UPSTREAM_WORKSPACE
+        # Handle multiple rosintall entries.
+        IFS=','  # Multiple URLs can be given separated by comma.
+        for rosinstall in $UPSTREAM_WORKSPACE; do
+            travis_run wstool merge $rosinstall
```
Comment (davetcoleman) [inline travis.sh]: supply the appropriate command line argument such that the easier rosinstall files listed are *overwritten* by later rosinstall files. currently travis is prompting the user and failing

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ script:
 - ROS_DISTRO: (required) which version of ROS i.e. kinetic
 - ROS_REPO: (default: ros-shadow-fixed) install ROS debians from either regular release or from shadow-fixed, i.e. http://packages.ros.org/ros-shadow-fixed/ubuntu
 - BEFORE_SCRIPT: (default: not set): Used to specify shell commands or scripts that run before building packages.
-- UPSTREAM_WORKSPACE (default: debian): When set as "file", the dependended packages that need to be built from source are downloaded based on a .rosinstall file in your repository. When set to a "http" URL, this downloads the rosinstall configuration from an http location
+- UPSTREAM_WORKSPACE (default: debian): When set as "file", the dependended packages that need to be built from source are downloaded based on a .rosinstall file in your repository. When set to a "http" URL, this downloads the rosinstall configuration from an http location. Multiple http entries can be given by separating each with comma.
```
Comment (davetcoleman) [inline README.md]: mention the ordering overwriting

File: .travis.yml [hunk 3]
```diff
@@ -15,6 +15,10 @@ env:
   matrix:
     - ROS_DISTRO=kinetic ROS_REPO=ros              UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'" TEST_BLACKLIST="moveit_core"
     - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'"
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall,https://raw.githubusercontent.com/ros-planning/moveit_docs/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'"
```
Comment (davetcoleman) [inline .travis.yml]: i don't like adding moveit_docs here - that repo is deprecated. perhaps we can just list the same moveit repo twice?

File: .travis.yml [hunk 4]
```diff
@@ -15,6 +15,10 @@ env:
   matrix:
     - ROS_DISTRO=kinetic ROS_REPO=ros              UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'" TEST_BLACKLIST="moveit_core"
     - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall BEFORE_SCRIPT="echo 'testing'"
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall,https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall  # Duplicated tests over the same .rosinstall file for testing purpose only.
+matrix:
+  allow_failures:
+    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall,https://raw.githubusercontent.com/ros-planning/moveit/kinetic-devel/moveit.rosinstall  # Duplicated tests over the same .rosinstall file for testing purpose only.
```
Comment (davetcoleman) [inline .travis.yml]: in fact it does on travis now! please remove

Comment (130s) [inline .travis.yml]: Addressed.

**Pull request merged**